crates/av/src/video_player.rs

```diff
@@ -105,9 +106,26 @@ fn play_videos(
                     sink.rate = Some(rate);
                 }
                 Ok(VideoData::Frame(frame, time)) => {
-                    last_frame_received = Some(frame);
+                    last_frame_received = Some(frame.data(0).to_vec());
```

Comment:
can we please make `last_frame_received` an enum to avoid copying the bytes of every frame (even if we don't display it) - i think frames may arrive in large batches if there's a network delay

Reply:
no it's fine